### PR TITLE
fix: readme, scripts, image push

### PR DESCRIPTION
**Pull request by psturc**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9282

## What
Small fixes/additions to readme & scripts

## Verification Steps
Verify that steps for setting up local environment are still valid by following the guide [here](https://github.com/aerogear/mobile-developer-console/pull/257/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,13 +80,7 @@ The mobile developer console will need to talk to the OpenShift API server direc
 
 To fix this, you will need to manually update the OpenShift master configuration to allow CORS requests from the mobile developer console.
 
-If you are using minishift, you should use the [CORS addon](https://github.com/minishift/minishift-addons/tree/master/add-ons/cors) to enable it.
-
-```
-git clone https://github.com/minishift/minishift-addons.git /tmp/
-minishift addons install /tmp/minishift-addons/add-ons/cors
-minishift addons apply cors
-```
```
Comment (psturc) [inline README.md]: This is automated now

File: deploy/deploy-image-stream.sh [hunk 2]
```diff
@@ -34,7 +34,6 @@ $OC_CMD patch -n $NAMESPACE deploymentconfig/mobile-developer-console --patch "$
 MDC_HOST=$($OC_CMD -n $NAMESPACE get route mobile-developer-console --template "{{.spec.host}}")
 $OC_CMD patch oauthclient/mobile-developer-console --patch "{\"redirectURIs\":[\"https://$MDC_HOST\"]}"
 
-$OC_CMD start-build -n $NAMESPACE mobile-developer-console
```
Comment (psturc) [inline deploy/deploy-image-stream.sh]: This command was triggering additional build, which is not needed.

File: Makefile [hunk 3]
```diff
@@ -39,19 +39,19 @@ docker_build:
...
 docker_push_release:
 	@docker login --username $(QUAY_USERNAME) --password $(QUAY_PASSWORD) ${SERVER}
-	docker push $(DOCKER_LATEST_TAG)
 	docker push $(DOCKER_RELEASE_TAG)
 
 .PHONY: docker_push_master
 docker_push_master:
 	@docker login -u $(QUAY_USERNAME) -p $(QUAY_PASSWORD) ${SERVER}
 	docker push $(DOCKER_MASTER_TAG)
+	docker push $(DOCKER_LATEST_TAG)
```
Comment (psturc) [inline Makefile]: to keep master with latest tag up-to-date

File: scripts/development.sh [hunk 4]
```diff
@@ -23,5 +23,6 @@ export NAMESPACE="$NAMESPACE"
 export ENABLE_BUILD_TAB="false"
 echo "Start local development server..."
 cd ../
+npm install
```
[the remark this answers is not in the document]
Comment (psturc) [reply scripts/development.sh]: I like this idea

File: README.md [hunk 5]
```diff
@@ -67,7 +66,7 @@ then you need to run the prepare script:
 and start the development server:
 
 ```
-export OPENSHIFT_HOST=<openshift-url>
+export OPENSHIFT_HOST=<openshift-hostname>
```
[the remark this answers is not in the document]
Comment (psturc) [reply README.md]: 👍  Fixed it where appropriate

**Comment by craicoverflow** [comment]
I've tested your new changes and can verify that `--skip-npm-install` works as expected.

**Pull request merged**